index.js

```diff
@@ -0,0 +1,7 @@
+var test = 0;
+
+const process = {
+  addTest(){
```

Comment:
'concise methods' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).